<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScoreLauncher</class>
 <widget class="QMainWindow" name="ScoreLauncher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="toolkits" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>561</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:bold 20px; background:white</string>
     </property>
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
    <widget class="QWidget" name="toolkitContents" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>561</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>459</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mouse Input</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="titleBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold 25px; </string>
    </property>
    <property name="plainText">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="composerBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:18px</string>
    </property>
    <property name="plainText">
     <string>composer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
